frontend/src/scenes/Login.js

```diff
@@ -90,6 +102,10 @@ export default function Login({ locale, routePath }) {
         >
           {STRINGS[locale].FACEBOOK_SIGN_UP}
         </FacebookButton>
+        <ActionLink variant="body1" role="button" onClick={()=>{window.location.href='https://m.me/callhomesg'}}>
```

Comment:
Use `<a>` and `href` for simple navigation. You can remove the `ActionLink` component if you do.